src/components/dashboard/Recovery/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Button, Card, Chip, Grid, Typography } from '@mui/material'
+import type { ReactElement } from 'react'
+
+import RecoveryLogo from '@/public/images/common/recovery.svg'
+import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { useDarkMode } from '@/hooks/useDarkMode'
+
+import css from './styles.module.css'
+
+export function Recovery(): ReactElement {
+  const isDarkMode = useDarkMode()
+
+  const onClick = () => {
+    // TODO: Open enable recovery flow
+  }
+
+  return (
+    <WidgetContainer>
+      <Typography component="h2" variant="subtitle1" className={css.label}>
+        New in {'Safe{Wallet}'}
```

Comment:
nit: The Figma design says `New in Safe` here

Reply:
The wording is incorrect in the design. This is a Safe{Wallet} feature.